src/store/soundsStore.js

```diff
@@ -29,8 +29,8 @@ const state = {
 }
 
 const getters = {
-	playSounds: (state) => {
-		if (state.userId) {
+	getPlaySounds: (state) => (userId) => {
+		if (userId) {
```

Comment:
Feels a bit weird to have to inject the current user (gives the wrong feeling you could read the setting of others), but seems like we can not access the state of other stores in here?

Reply:
you can: see https://vuex.vuejs.org/guide/modules.html#namespacing

then something like `rootGetters.actorStore.getUser()` or `rootGetters['actorStore/getUser']` might work

Reply:
since we didn't give modules a name, maybe even `rootGetters.getUser()` might work